app/src/main/java/com/pluu/webtoon/ui/episode/EpisodeFragment.kt

```diff
@@ -1,41 +1,33 @@
-package com.pluu.webtoon.ui
+package com.pluu.webtoon.ui.episode
 
 import android.app.Activity
 import android.app.ProgressDialog
 import android.content.Intent
 import android.content.res.Configuration
 import android.os.Bundle
-import android.util.Log
-import android.view.LayoutInflater
-import android.view.View
-import android.view.ViewGroup
+import android.view.*
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Pluu/WebToon/blob/ae64ac2267c99cb98c2fd2ab1b680508d11b226a/app/src/main/java/com/pluu/webtoon/ui/episode/EpisodeFragment.kt#L8"></span>Wildcard import (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>